app/views/catalog/_document.html.erb

```diff
@@ -9,9 +9,11 @@
     <%= render_document_partial document, :index %>
   </div>
   <div class="col-sm-2">
-  	  <%# bookmark functions for items/docs -%>
-	  <%= render_index_doc_actions document, :wrapping_class => "select-item" %>
-	  <div class="bookcover" id="OCLC:<%= bookcover_oclc(document) %>" data-oclc="<%= bookcover_oclc(document) %>">
-	  </div>
+    <%# bookmark functions for items/docs -%>
+  	<%= render_index_doc_actions document, :wrapping_class => "select-item" %>
+    <% if document['oclc_id_display'].present? %>
```

Comment:
I'd recommend consolidating the three references to the OCLC number here ... maybe something like:
` oclc_num = bookcover_oclc(document)`
`<div class="bookcover" id="OCLC:<%= oclc_num %>" data-oclc="<%= oclc_num %>">`

In the helper function, I think the redundant (first) `bookcover_oclc` method can be safely deleted. I haven't checked to see whether there are any references to `bookcover_isbn`, but if you're confident that there aren't, I'd suggest commenting out that method and waiting to see if it causes any unexpected disasters.

Reply:
Code review is awesome. Thanks to you both! I'll try this.

Reply:
I made the updates and went ahead and merged into dev after the tests passed. Hopefully that didn't break anything! Jim, I'll make you an issue for book cover tests.
https://github.com/cul-it/blacklight-cornell/pull/2021/commits/71cf85604618f203212be7fec56fac7f1b48f4dc